packages/blade/src/components/QuickFilters/types.ts

```diff
@@ -0,0 +1,58 @@
+import type { DataAnalyticsAttribute, TestID } from '~utils/types';
+
+type QuickFilterGroupProps = {
+  /*
+   onChange is a function that is called when the selected quick filter changes.
+   it returns an object with either value or values.
+   values is an array of strings that are the selected quick filters if the selectionType is 'multiple'.
+   value is a string that is the selected quick filter if the selectionType is 'single'. 
+*/
+  onChange?: (params: { values?: string[] | string }) => void;
```

Comment:
This is not same as ChipGroup. 

I think it's best to keep this API 1:1 with the ChipGroup for familiarity, QuickFilters are basically ChipGroups with different styling.

Reply:
done